plugins/flytekit-aws-sagemaker/flytekitplugins/awssagemaker_inference/boto3_agent.py

```diff
@@ -54,9 +57,20 @@ async def do(self, task_template: TaskTemplate, inputs: Optional[LiteralMap] = N
             inputs=inputs,
         )
 
-        outputs = None
+        outputs = {"result": {"result": None}}
```

Comment:
Isn't None Json serializable? could we create a dummy class here

<img width="318" alt="Screenshot 2024-05-08 at 7 12 33 PM" src="https://github.com/flyteorg/flytekit/assets/37936015/1faa2af3-b6e5-4d91-aff8-362c5c826786">

Reply:
modified it to `None`